File: tmp/check-pg.mjs
import { Client } from 'pg';
const client = new Client({ host: 'localhost', port: 5432, database: 'jesse_db', user: 'jesse_user', password: 'password' });
await client.connect();
const res = await client.query("SELECT id, status, exchange, symbol, timeframe, strategy, started_at, finished_at FROM backtestsession ORDER BY started_at DESC LIMIT 5");
console.log(JSON.stringify(res.rows, null, 2));
await client.end();

File: tmp/check-pg.py
import psycopg2, json
conn = psycopg2.connect(host="localhost", port=5432, database="jesse_db", user="jesse_user", password="password")
cur = conn.cursor()
cur.execute("SELECT id, status, metrics, exception, execution_duration, created_at FROM backtestsession ORDER BY created_at DESC LIMIT 5")
rows = cur.fetchall()
for r in rows:
    sid = str(r[0])
    status = r[1]
    exc = r[3]
    dur = r[4]
    created = str(r[5])
    metrics = json.loads(r[2]) if r[2] else None
    print(f"id={sid[:12]} status={status} dur={dur} exc={exc} created={created}")
    if metrics:
        print(f"  metrics keys: {list(metrics.keys())[:10]}")
cur.close()
conn.close()